pkg/aws/helper.go

```diff
@@ -417,8 +369,8 @@ func (h *AWSHelper) EnsureListener(lbARN, tgARN string, port int, udp bool) erro
 		}
 	}
 	protocol := elbv2.ProtocolEnumTcp
-	if udp {
-		protocol = "UDP"
+	if tcp {
```

Comment:
i think we can keep this `if udp` test b/c it already defaults to TCP already in the line right above, so as long as we call EnsureListener with param `udp == false` this function will continue to work (need to update the EnsureListener call for the VPN in `pkg/aws/install.go` with `udp` set to `false` since we're now using TCP.

or we can remove the `udp` parameter from the function entirely (always assuming TCP), and remove this `if udp` test, and update all the EnsureListener callers.

Reply:
We're about to remove the need for this code, so I don't think we should bother changing this.